OrbitGl/App.cpp

```diff
@@ -239,6 +244,27 @@ void OrbitApp::PostInit() {
 
   string_manager_ = std::make_shared<StringManager>();
   GCurrentTimeGraph->SetStringManager(string_manager_);
+
+  if (absl::GetFlag(FLAGS_enable_tracepoint_service)) {
+    tracepoint_manager_ = TracepointServiceClient::Create(grpc_channel_);
+
+    thread_pool_->Schedule([this] {
+      tracepoint_manager_->PopulateWithServerData();
+      ErrorMessageOr<std::vector<TracepointInfo>> result = tracepoint_manager_->GetTracepointList();
```

Comment:
+1 
This would also eliminate the need for storing the tracepoints in the tracepoint_manager_. And you can also remove tracepoint_manager_ from App.h, you can have it as a local variable here.

Reply:
Sure. Thanks for the idea! I see how is was redundant.